jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiController.java

```diff
@@ -88,6 +92,29 @@ public ApiResponse<T> update(@RequestParam UUID tenantId,
     return new ApiResponse<>(service.updateResource(entity));
   }
 
+  public ApiResponse<T> patch(@RequestParam UUID tenantId,
+                               @RequestBody List<Object> body) {
+
+    var ops = readPatchOperationsFromPayload(body);
+
+    var entities = new ArrayList<T>();
+
+    for (PatchOperation<T> patchOperation : ops) {
```

Comment:
Is this worth parallelising? I'm just thinking about the payload we may run through this in accruals. We'd need a max parallel threads if we did and to think about how the exception is thrown. This could be a premature optimisation at the moment so happy to part this on the scratchpad or to ignore for now.